modules/ai/assets/js/editor/utils/promotion-introduction-session-validator.js

```diff
@@ -1,24 +1,3 @@
-export const HOURS_BETWEEN_PROMOTION_INTRODUCTIONS = 48;
-const HOURS_DELTA_IN_MILLISECONDS = HOURS_BETWEEN_PROMOTION_INTRODUCTIONS * 60 * 60 * 1000;
-
-function passedRequiredHoursSinceLastPromotion( currentTimestamp, lastPromotionTimestamp ) {
-	return currentTimestamp - parseInt( lastPromotionTimestamp ) >= HOURS_DELTA_IN_MILLISECONDS;
-}
-
-export function shouldShowPromotionIntroduction( session ) {
-	const editorSessionValue = session.getItem( 'ai_promotion_introduction_editor_session_key' );
-	const lastPromotionTimestamp = editorSessionValue?.split( '#' )[ 1 ];
-
-	if ( editorSessionValue === window.EDITOR_SESSION_ID ) {
-		return false;
-	}
-
-	const currentTimestamp = new Date().getTime();
-
-	if ( ! editorSessionValue || ! lastPromotionTimestamp || passedRequiredHoursSinceLastPromotion( currentTimestamp, lastPromotionTimestamp ) ) {
-		session.setItem( 'ai_promotion_introduction_editor_session_key', `${ window.EDITOR_SESSION_ID }#${ currentTimestamp }` );
-		return true;
-	}
-
+export function shouldShowPromotionIntroduction( ) {
```

Comment:
I'm not removing all the PLGs, but only the code that triggers them so that it will always return false